bitrise.yml

```diff
@@ -0,0 +1,63 @@
+---
+format_version: '13'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+
+trigger_map:
+- push_branch: main
+  workflow: build
+- pull_request_source_branch: "*"
+  workflow: build
+
+app:
+  envs:
+  - BITRISE_PROJECT_PATH: Backyard Birds.xcodeproj
+  - BITRISE_SCHEME: Backyard Birds
+  - BITRISE_DISTRIBUTION_METHOD: development
+
+meta:
+  bitrise.io:
+    stack: osx-xcode-15.3.x
+    machine_type_id: g2-m1.8core
+
+workflows:
+  build:
+    summary: Build your Xcode project.
+    description: The workflow will first clone your Git repository, cache and install
+      your project's dependencies if any and build your project.
+    steps:
+    - activate-ssh-key@4: {}
+    - git-clone@8: {}
+    - restore-spm-cache@2: {}
+    - xcode-test@5:
+        inputs:
+        - xcodebuild_options:
+        - test_plan: UnitTests
+    - save-spm-cache@1: {}
+    - deploy-to-bitrise-io@2: {}
+
+  archive_and_export_app:
+    summary: Create an IPA file to install your app on a device or share it with your
+      team.
+    description: The workflow will first clone your Git repository, cache and install
+      your project's dependencies if any, export an IPA file from the project and
+      save it.
+    steps:
+    - activate-ssh-key@4: {}
```

Comment:
this comment applies to the other workflow - we should use Github App, not SSH

Reply:
Actually, I'm having trouble getting webhooks set up with Github App, so maybe best to stick with OAuth for now